Common/Tests/Utilities/FileUtils.cs

```diff
@@ -57,8 +69,13 @@ where f.StartsWith(destDir)
                 var copyFrom = Path.Combine(sourceDir, newFile);
                 var copyTo = Path.Combine(destDir, newFile);
                 try {
-                    File.Copy(copyFrom, copyTo);
-                    File.SetAttributes(copyTo, FileAttributes.Normal);
+                    if (copyTo.Length < MaxFileNameLength && copyFrom.Length < MaxFileNameLength) {
+                        var copyToDir = Path.GetDirectoryName(copyTo);
+                        if (copyToDir.Length < MaxDirectoryLength) {
+                            File.Copy(copyFrom, copyTo);
+                            File.SetAttributes(copyTo, FileAttributes.Normal);
+                        }
+                    }
                 } catch {
                     Debug.WriteLine("Failed to copy " + copyFrom + " to " + copyTo);
```

Comment:
We should still print the debug message (here and everywhere else we used to print) if we hit max_path since the exception is longer getting caught